Tests/AppTests/WebpageSnapshotTests.swift

```diff
@@ -0,0 +1,20 @@
+@testable import App
+
+import SnapshotTesting
+import XCTest
+
+
+let defaultSize = CGSize(width: 800, height: 600)
+
+class WebpageSnapshotTests: XCTestCase {
+    func test_home() throws {
+        // record = true
```

Comment:
Uncommenting this will re-record the snapshots, overwriting the files in `__Snapshots__` (see below) _and_ mark the test as failed (just so you don't accidentally and silently change the expectations without review).

Setting the test to record while tweaking the design is a nice way to check design changes against the committed baseline in a local diff viewer.